File: ReceiptRewards.WebServices/src/com/model/Reward.java
package com.model;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class Reward {
    
    private int rewardId;
    private Code code;
    private Form form;

    public Reward() {
        this.rewardId = 0;
        this.code = null;
        this.form = new Form();
    }

    public Reward(int rewardId, Code code, Form form) {
        this();
        this.rewardId = rewardId;
        this.code = code;
        this.form = form;
    }

    public int getRewardId() {
        return rewardId;
    }

    public void setRewardId(int rewardId) {
        this.rewardId = rewardId;
    }

    public Code getCode() {
        return code;
    }

    public void setCode(Code code) {
        this.code = code;
    }

    public Form getForm() {
        return form;
    }

    public void setForm(Form form) {
        this.form = form;
    }
    
    @Override
    public String toString() {
        // TODO: Implement
        return "";
    }
}

File: ReceiptRewards.PCL/Models/Reward.cs
using System;

namespace ReceiptRewards.PCL.Models {

    public class Reward {

        private int _rewardId;
        private Code _code;
        private Form _form;


        public int rewardId {
            get { return _rewardId; }
            set { _rewardId = value; }
        }

        public Code code {
            get { return _code; }
            set { _code = value; }
        }

        public Form form {
            get { return _form; }
            set { _form = value; }
        }

    }
}